.github/workflows/sdkv2_deprecation.yaml

```diff
@@ -1,5 +1,10 @@
 name: SDKv2 Deprecation Check
 
+permissions:
+  contents: write
```

Comment:
does this need `contents: write`? probably just `read`, unless i've overlooked something

Reply:
I think you're right. In fact I think we can remove permissions here since the job already specifies permissions.